src/docs/releases/2.0.0.md

```diff
@@ -1,4 +1,4 @@
-# Orchard Core 2.0.0
+ore 2.0.0
```

Comment:
Ore? Brothers of the mine rejoice! https://www.youtube.com/watch?v=ytWz0qVvBZ0

```suggestion
Orchard Core 2.0.0
```

Reply:
Oops :)